interpreter/src/main/scala/pl/touk/nussknacker/engine/testmode/TestDataPreparer.scala

```diff
@@ -1,53 +1,26 @@
 package pl.touk.nussknacker.engine.testmode
 
-import pl.touk.nussknacker.engine.ModelData
 import pl.touk.nussknacker.engine.api.NodeId
-import pl.touk.nussknacker.engine.api.process.{ComponentUseCase, SourceTestSupport}
-import pl.touk.nussknacker.engine.api.test.TestData
-import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
-import pl.touk.nussknacker.engine.compile.ExpressionCompiler
-import pl.touk.nussknacker.engine.compile.nodecompilation.NodeCompiler
-import pl.touk.nussknacker.engine.graph.node.SourceNodeData
-import pl.touk.nussknacker.engine.resultcollector.PreventInvocationCollector
-import pl.touk.nussknacker.engine.spel.SpelExpressionParser
+import pl.touk.nussknacker.engine.api.process.{Source, SourceTestSupport}
+import pl.touk.nussknacker.engine.api.test.{ScenarioTestData, TestRecord}
 
 case class ParsedTestData[T](samples: List[T])
 
 object TestDataPreparer {
 
-  def prepareDataForTest[T](sourceTestSupport: SourceTestSupport[T], testData: TestData): ParsedTestData[T] = {
+  def prepareDataForTest[T](sourceTestSupport: SourceTestSupport[T], scenarioTestData: ScenarioTestData, sourceId: NodeId): ParsedTestData[T] = {
```

Comment:
I wonder, this method (and ParsedTestData) is used only for Flink. Is it worthy to keep it in interpreter?

Reply:
You're right, I removed `TestDataPreparer` and moved specific methods to Flink and Lite modules.